File: VB/DxBlazorApplication1/Services/DataHelpers/LabelDataHelper.vb
Imports DxBlazorApplication1.Models
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Text.Json
Imports System.Threading.Tasks

Namespace DxBlazorApplication1.Services.DataHelpers

    Public Class LabelDataHelper

        Public Shared Function GetLabels() As List(Of LectureType)
            Return lectureTypes
        End Function

        Private Shared ReadOnly lectureTypes As List(Of LectureType) = New List(Of LectureType)() From {New(_, _) With {.Id = 1, .Name = "Lecture", .Color = Color.Coral, .BackgroundCssClass = "label-1", .TextCssClass = "label-1"}, New(_, _) With {.Id = 2, .Name = "Seminar", .Color = Color.ForestGreen, .BackgroundCssClass = "label-2", .TextCssClass = "label-2"}, New(_, _) With {.Id = 3, .Name = "Laboratory", .Color = Color.LightSkyBlue, .BackgroundCssClass = "label-3", .TextCssClass = "label-3"}}
    End Class
End Namespace

File: CS/DxBlazorApplication1/Services/DataHelpers/LabelDataHelper.cs
using DxBlazorApplication1.Models;
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Text.Json;
using System.Threading.Tasks;

namespace DxBlazorApplication1.Services.DataHelpers {
    public class LabelDataHelper {
        public static List<LectureType> GetLabels() => lectureTypes;

        private static readonly List<LectureType> lectureTypes = new List<LectureType>() {
            new(){ Id = 1, Name = "Lecture", Color = Color.Coral, BackgroundCssClass = "label-1", TextCssClass = "label-1" },
            new(){ Id = 2, Name = "Seminar", Color = Color.ForestGreen, BackgroundCssClass = "label-2", TextCssClass = "label-2" },
            new(){ Id = 3, Name = "Laboratory", Color = Color.LightSkyBlue, BackgroundCssClass = "label-3", TextCssClass = "label-3" },
        };
    }
}
